packages/cysync-core/src/context/appUpdate.tsx

```diff
@@ -71,6 +71,15 @@ export const AppUpdateProvider: React.FC<AppUpdateProviderProps> = ({
     ],
   );
 
+  useEffect(() => {
+    const minutes = 15;
+    const checkUpdateInterval = setInterval(
+      checkForUpdates,
+      minutes * 60 * 1000,
+    );
+    return () => clearInterval(checkUpdateInterval);
+  }, [checkForUpdates]);
```

Comment:
If useEffect depends on a function, we need to make sure that function does not change on re-render. Use `useCallback` for that.

Reply:
Done